proxy/src/main.rs

```diff
@@ -1,20 +1,22 @@
-use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+use secstr::SecUtf8;
+use std::convert::TryFrom;
 
-use librad::keys;
-use librad::net::{self, discovery};
 use librad::paths;
-use librad::peer;
+use radicle_keystore::pinentry::{self, Pinentry as _};
 
 use proxy::coco;
 use proxy::env;
 use proxy::http;
+use proxy::keystore;
 use proxy::registry;
 
 /// Flags accepted by the proxy binary.
 struct Args {
     /// Host name or IP for the registry node to connect to. If the special value "emulator" is
     /// provided the proxy will not connect to a node but emulate the chain in memory.
     registry: String,
+    /// Choosing a base path for Peer configuration. Uses a default location if not provided.
+    path: Option<std::path::PathBuf>,
```

Comment:
The param itself is quite generally named and doesn't really communicate the function. Also we don't really have a use for it at the moment, so I'd drop it until we need it.